lib/components/Article.js

```diff
@@ -55,7 +55,7 @@ Article.propTypes = {
 	})
 };
 
-// this fn comp. is now return Article and passing same props to Article and also context object as props to Article
+// this fn comp. is now return Article and passing same props to Article and also context object as props to Article while problem with this is that we can control generic component which could be used for same purpose to get the data from ContextApi

```

Comment:
this is related to #21 should cherry pick it as Patch

Reply:
OK! this should be approved as can be done prior to this PR